### PR TITLE
Prepare Release v1.1.0

### PR DESCRIPTION
**Pull request by giladgray**
:fireworks: __Highlights__: BSD License, `@import "~package"` support, mobile-friendlier docs, so many bug fixes 

:book: __Latest docs__: [blueprintjs.com/docs](http://blueprintjs.com/docs)

## General

- :star: __Updated license__ to BSD-3-Clause to clarify patent license similar to Palantir’s [AtlasDB open source project](https://github.com/palantir/atlasdb) #209, #225 
- :star: Sass files support `@import "~package"` syntax a la Webpack's sass-loader for `node_modules` resolution! #123, #211
- clean up a number of package dependencies so they're more accurate #227 

## @blueprintjs/core `1.1.0`

- See general notes above
- __Improved__ added more constants to `Classes`: INLINE, FILL, LABEL, SELECT #134
- __Fixed__ Hotkey Dialog now appears only once with all hotkeys #142
- __Fixed__ hotkeys can be triggered on checkbox and radio inputs #145
- __Fixed__ readability of links in intent'ed Toasts #172
- __Fixed__ Portals no longer add additional space at the bottom of the page #147
- __Fixed__ text color of dark `select` dropdowns on Windows #138
- __Removed__ company-affiliated icons #208 
  - if you relied on these icons, they can be found easily in other open-source fonts.

## @blueprintjs/datetime `1.1.0`

- See general notes above
- __Fixed__ text color of dark month/year dropdowns on Windows #138

## @blueprintjs/table `1.1.0`

- See general notes above

## Documentation

- :star: __Improved__ basic mobile support! not perfect but at least usable. #105, #128
- __Improved__ refactored docs layout for better usability. PageUp/Down work as expected! #149, #89, #122
- __Fixed__ favicons on landing and docs pages #92
- __Fixed__ intent dropdowns update correctly #154
- __Fixed__ removed version selector as there's only going to be one for some time #129
- __Changed__ click the docs logo to jump to the Overview page #93

### CODE REVIEW

**Comment by adidahiya** [comment]
trying out an idea to fix the build: https://github.com/palantir/blueprint/pull/227

**Comment by giladgray** [comment]
did a rebuild without cache cuz package.jsons have changed significantly.
seems to be progressing smoothly now...

File: docs/docs/docs.css [hunk 1]
```diff
@@ -5236,6 +5232,8 @@ table.pt-table.pt-interactive tbody tr:hover td {
     border-bottom-color: rgba(16, 22, 26, 0.4); }
 .pt-dark .pt-datepicker .pt-datepicker-caption select {
       color: #f5f8fa; }
+.pt-dark .pt-datepicker .pt-datepicker-caption option {
+      background-color: #30404d; }
```
Comment (giladgray) [inline docs/docs/docs.css]: 🆒 it is pretty nifty getting this diff on the compiled CSS!

**Comment by adidahiya** [comment]
:shipit:

**Comment by giladgray** [comment]
one last thing: it still says `v1.0.1` in the navbar because there's no 1.1.0 tag yet. added a little snippet to the gulp task so it'll include package.json version when computing the list.

**Pull request merged**